aspnet/mvc/overview/getting-started/database-first-development/publish-to-azure.md

```diff
@@ -1,129 +1,111 @@
+> [!div class="checklist"]
+> * Deploy your web app on Azure
+> * Publish the database to SQL Azure
 
-Select Microsoft Azure Websites.
+## Prerequisites
 
-![select Azure](publish-to-azure/_static/image2.png)
+* [Enhance data validation](enhancing-data-validation.md)
 
-If you are not signed in to Azure, provide your Azure account credentials. Then, select New to create a new web app.
+## Deploy your web app on Azure
 
-![new site](publish-to-azure/_static/image3.png)
+You need an Azure account to complete this tutorial:
 
-Create a unique name for your web app. You will know the name is unique if you see a green check mark to the right of the name field. Select a region for your web app. Select **Create new server** for the database, and provide a user name and password for this new database server. When finished, click **Create**.
+- You can [open an Azure account for free](https://azure.microsoft.com/pricing/free-trial/?WT.mc_id=A261C142F) - You get credits you can use to try out paid Azure services, and even after they're used up you can keep the account and use free Azure services.
+- You can [activate MSDN subscriber benefits](https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/?WT.mc_id=A261C142F) - Your MSDN subscription gives you credits every month that you can use for paid Azure services.
 
-![create site](publish-to-azure/_static/image4.png)
+To publish your web app, right-click the **ContosoSite** project and select **Publish**. Click **Start**.
 
-Your connection values are now all set. You can leave these values unchanged.
+In **Pick a publish target**, select **App Service** > **Create New** and then select **Publish**.
 
-![connection](publish-to-azure/_static/image5.png)
+![Screenshot of Pick a publish target dialog.](publish-to-azure/_static/image2.png)
 
-Click **Next**.
+If you are not signed in to Azure, provide your Azure account credentials. **Create App Service** appears.
 
-For Settings, notice that two database connections are specified - ApplicationDBContext and ContosoUniversityDataEntities. ApplicationDBContext is the connection for user account tables. These values only show the connection strings for the databases. It does not mean that these databases will be published when you publish your site. You will publish your database project after you have finished publishing the web app.
+![Screenshot of Create App Service dialog.](publish-to-azure/_static/image3.png)
 
-![](publish-to-azure/_static/image6.png)
+Create a unique name for your web app. You will know the name is not unique if you see this error message: *Name is not available, please choose another*. Select your subscription and then either select an existing resource group and hosting plan, or create new ones.
 
-The ellipsis (...) next to the database connection shows you the details of the connection string. Click the ellipsis next to ContosoUniversityDataEntities.
+Select **Create a SQL Database**.
```

Comment:
The image3.png should go after this instruction since that's what the callout calls attention to.